### DISCUSSION TITLE
Why does Discord no longer exist?

### DISCUSSION THREAD

**Discussion by uilianries** [taocpp/taopq#66] [Q&A] [answered]
Checking my Discord list, I noted the TaoCpp no longer exist and those links related to the server were removed. Any particular reason for giving up? By my experience, I know managing Github issues, Github discussions, email and chat server can be too much, some days you barely have time to write some code.

**Comment by d-frey** [comment, marked as answer]
It seemed unused and therefore we decided to remove it. GitHub issues and discussions are enough.

**Reply by ColinH** [reply]
If I remember correctly we set up Discord _before_ GitHub Discussions were introduced because we wanted something besides Issues, which we've now got right here.

**Reply by uilianries** [reply]
Makes sense, as all discussions will be centralized and public. Thank you for your quick answer!
